From c74b360ba5a3290cdd99e7dac203a6a2de750a3c Mon Sep 17 00:00:00 2001
From: Rainer Sigwald <raines@microsoft.com>
Date: Fri, 8 Sep 2023 19:41:06 +0000
Subject: [PATCH 1/2] Eliminate prebuilts

---
 Directory.Build.props | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/Directory.Build.props b/Directory.Build.props
index b7aa431..f075a6a 100644
--- a/Directory.Build.props
+++ b/Directory.Build.props
@@ -27,11 +27,6 @@
     <RestoreUseStaticGraphEvaluation>true</RestoreUseStaticGraphEvaluation>
   </PropertyGroup>
 
-  <ItemGroup>
-    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All"/>
-    <PackageReference Include="Nerdbank.GitVersioning" Version="3.6.133" PrivateAssets="all" />
-  </ItemGroup>
-
   <ItemGroup>
     <None Include="$(MSBuildThisFileDirectory)\branding\MSBuild-NuGet-Icon.png"
           Pack="true"
-- 
2.42.0

