From 8aa9fe051882d9581df936828070c9ddd099bc88 Mon Sep 17 00:00:00 2001
From: Matt Mitchell <mmitche@microsoft.com>
Date: Tue, 18 Jul 2023 12:29:29 -0700
Subject: [PATCH] Eliminate prebuilts

---
 .props/Product.props | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/.props/Product.props b/.props/Product.props
index f81957c0..32475340 100644
--- a/.props/Product.props
+++ b/.props/Product.props
@@ -27,17 +27,6 @@
       <PrivateAssets>All</PrivateAssets>
     </PackageReference>
   </ItemGroup>
-
-  <ItemGroup>
-    <!--Build Infrastructure-->
-    <PackageReference Include="Microsoft.VisualStudioEng.MicroBuild.Core" Version="0.4.1">
-      <PrivateAssets>All</PrivateAssets>
-    </PackageReference>
-  </ItemGroup>
-
-  <ItemGroup>
-    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />
-  </ItemGroup>
   
   <PropertyGroup>
     <!-- Enable all the latest CA rules from 'Microsoft.CodeAnalysis.NetAnalyzers' as build warnings by default -->
-- 
2.41.0.windows.2

