From 152b8a363213133fc3ec87e3abd6595ac98f0b57 Mon Sep 17 00:00:00 2001
From: Jan Jones <janjones@microsoft.com>
Date: Mon, 7 Jul 2025 11:04:42 +0000
Subject: [PATCH] Remove PolySharp

---
 src/Spectre.Console/Spectre.Console.csproj | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/src/Spectre.Console/Spectre.Console.csproj b/src/Spectre.Console/Spectre.Console.csproj
index bb56ec6..ddfa90e 100644
--- a/src/Spectre.Console/Spectre.Console.csproj
+++ b/src/Spectre.Console/Spectre.Console.csproj
@@ -22,10 +22,6 @@
     <PackageReference Include="Wcwidth.Sources">
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
-    <PackageReference Include="PolySharp">
-      <PrivateAssets>all</PrivateAssets>
-      <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
-    </PackageReference>
   </ItemGroup>
 
   <PropertyGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
-- 
2.34.1

